services/clusters-keeper/Makefile

```diff
@@ -3,3 +3,25 @@
 #
 include ../../scripts/common.Makefile
 include ../../scripts/common-service.Makefile
+
+.env: .env-devel ## creates .env file from defaults in .env-devel
```

Comment:
MINOR: perhaps we ca have this in `common-service.Makefile`? I am sure it will be useful in many places

Reply:
that might be. will try to move this in the next PR